nodeup/pkg/model/cloudconfig.go

```diff
@@ -33,6 +34,34 @@ const (
 	VM_UUID_FILE_PATH = "/etc/vmware/vm_uuid"
 )
 
+type azureCloudConfig struct {
```

Comment:
Some of these look like effectively renamed properties used by multiple cloud providers already.
E.g VMType = MachineType = InstanceType?

We would prefer to map common names if possible.

Reply:
Ah these have to be same as the config described in https://kubernetes-sigs.github.io/cloud-provider-azure/install/configs/

For example, `vmType` is either `vmss` or `standard` and controls the type of nodes. When this is set to `vmss`, nodes are created with Virtual Machine Scale Set, which is equivalent to AWS Auto Scaling Group. When this is set to `standard`, just Virtual Machines (like EC2 instance) are used.

I've added a link to the page.